db/interfaces/interfaces.go

```diff
@@ -0,0 +1,19 @@
+package interfaces
+
+import "github.com/livepeer/leaderboard-serverless/models"
+
+type DB interface {
+	InsertStats(stats *models.Stats) error
+	AggregatedStats(query *models.StatsQuery) (*models.AggregatedStatsResults, error)
+	MedianRTT(query *models.StatsQuery) (float64, error)
```

Comment:
What is `MedianRTT`, did we always have it?

Reply:
No, it is new and necesasry to calculate the AI scores.  The AI scores can't be benchmarked against segments like in transcoding.  As a result, we use median RTT across all Orchestrators to determine individuals performance on the network.